common/utils/util.js

```diff
@@ -153,16 +147,19 @@ const processReplacements = (input, replacementDetails) => {
   return JSON.parse(newInput)
 }
 
-const formatDateWith = pattern => isoString => {
```

Comment:
Feels terrible, but we are missing the timezone when dates are returned from the API. This means that accurate test data locally and remotely (in CI) is causing issues when testing DST is getting correctly applied.. 🤔